packages/core/contracts/financial-templates/perpetual-multiparty/PositionManager.sol

```diff
@@ -243,7 +223,7 @@ contract PositionManager is FeePayer, AdministrateeInterface {
     /**
      * @notice Cancels a pending transfer position request.
      */
-    function cancelTransferPosition() external onlyPreExpiration() nonReentrant() {
+    function cancelTransferPosition() external notEmergencyShutdown() nonReentrant() {
```

Comment:
Should we enable transfer requests to be cancelled during an emergency shutdown? Shouldn't it be just as flexible as `cancelWithdrawal` which is allowed during a shutdown?

Reply:
In fact I think both `cancelTransferPosition` and `cancelWithdrawal` should be blocked if the contract is an emergency shutdown. if there is a shutdown ALL functionality should be blocked IMHO. I have updated the contracts & tests accordingly.